cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -488,6 +479,56 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             };
         }
     });
+    AccessEditor = AbstractEditor.extend({
+        templateName: 'access-editor',
+        className: 'edit-settings-access',
+        events : {
+            'change #prereq': 'handlePrereqSelect',
```

Comment:
As a general note, we prefer classes to ids (because of ids can cause non-uniqueness issue). Since this is a modal, it is extremely unlikely 2 will ever be shown at the same time, but in the future please use classes instead of ids.